.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      OS: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+
+      - name: Get dependencies
```

Comment:
since we're `vendor/`'ing dependencies in source control, do we need to do a `go get` at all?

from what I remember in https://github.com/opencontrol/compliance-masonry/pull/355 (or at least think I remember, who knows), in go 1.14+, `go` commands will assume `-mod=vendor` by default if the `vendor/` directory exists; so like in `go test`, `go build`, all that good stuff, it should use the `vendor/` dir by default